# -*- coding: utf-8 -*-
"""
Created on Fri Nov 25 11:27:39 2022

@author: md arman hossain
"""

import sys

from PyQt5.QtWidgets import (
    QApplication, QDialog,QWidget, QMainWindow, QMessageBox,
)
from PyQt5.uic import loadUi

from graph import Window as GraphView

class Excel(QMainWindow):
    def __init__(self, parent=None):
        super().__init__(parent)
        loadUi("mainwindow.ui", self)
        self.initvar()
        self.connect()
        
    def initvar(self):
        self.graphview = GraphView()
        
    def connect(self):
        self.actionGraphical.triggered.connect(self.actiongraphicaltriggered)
        self.actionEPSS.triggered.connect(self.actionepsstriggered)
        self.actionExport.triggered.connect(self.actionexportfiletriggered)
        
    def actiongraphicaltriggered(self):
        self.graphview.show()
        
    def actionepsstriggered(self):
        pass
    
    def actionexportfiletriggered(self):
        pass
        
if __name__ == "__main__":
    app = QApplication(sys.argv)
    win = Excel()
    win.show()
    sys.exit(app.exec())